website/content/v1.10/introduction/getting-started.md

```diff
@@ -1,323 +1,229 @@
+Boot your hardware using the ISO image you just downloaded. At this stage, you'll:
 
-## Define the Kubernetes Endpoint
+- Boot one machine as your control plane node.
+- Boot additional machines as worker nodes (this is optional).
 
-In order to configure Kubernetes, Talos needs to know
-what the endpoint of the Kubernetes API Server will be.
+You’ll see the Talos dashboard once your hardware boots from the ISO image.
 
-Because we are only creating a single control plane node in this guide, we can use the control plane node directly as the Kubernetes API endpoint.
+**Note**: The ISO runs entirely in RAM and won't modify your disks until you apply a configuration**.**
 
-Identify the IP address or DNS name of the control plane node that was booted above, and convert it to a fully-qualified HTTPS URL endpoint address for the Kubernetes API Server which (by default) runs on port 6443.
-The endpoint should be formatted like:
+**Troubleshooting network connectivity:** If your machine fails to establish a network connection after booting, you may need to add network drivers through system extensions. Add these extensions to your Talos image via the [Image factory](https://factory.talos.dev/), or see the [system extensions repository](https://github.com/siderolabs/extensions) for more information.
 
-- `https://192.168.0.2:6443`
-- `https://kube.mycluster.mydomain.com:6443`
+## Step 3: Store Your Node IP Addresses in a Variable
 
-> NOTE: For a production cluster, you should have three control plane nodes, and have the endpoint allocate traffic to all three - see [Production Notes]({{< relref "prodnotes#control-plane-nodes" >}}).
+To create variables for your machines’ IP addresses:
 
-## Configure Talos Linux
 
-When Talos boots without a configuration, such as when booting off the Talos ISO, it
-enters maintenance mode and waits for a configuration to be provided.
+1. Copy the IP address displayed on each machine console, including the control plane and any worker nodes you’ve created. 
 
-> NOTE: Talos initially loads the OS to RAM, and only installs to disk after the configuration is applied.
-> If you reboot the machine before applying machine config, make sure your boot media is still present.
+If you don’t have a display connected, retrieve the IP addresses from your DHCP server.
 
-Unlike traditional Linux, Talos Linux is _not_ configured by SSHing to the server and issuing commands.
-Instead, the entire state of the machine is defined by a `machine config` file which is passed to the server.
-This allows machines to be managed in a declarative way, and lends itself to GitOps and modern operations paradigms.
+![](https://lh7-rt.googleusercontent.com/docsz/AD_4nXf8GSw-N_xtOrf9SF5woCrykShYdR4viP50-5C0vrtAGwe6Y7s7W4nzMPFrRI7DSLGXRhnDjYZB-nDzSW1ekiVbzgr-mYPEjfH22nFQgwAFBE-HcdV_ZahOoZyg6qfFP184Yo0_MA?key=EnbElPOmPgqlw8y3xF7EsA)
```

Comment:
what is that? images should be part of the docs (we should put them to the git repo)